core/commands/add.go

```diff
@@ -279,7 +286,13 @@ You can now check what blocks have been created by:
 		fileAdder.Silent = silent
 		fileAdder.RawLeaves = rawblks
 		fileAdder.NoCopy = nocopy
-		fileAdder.CidBuilder = &prefix
+		fileAdder.CidBuilder = prefix
+
+		if inline {
+			fileAdder.CidBuilder = cidutil.InlineBuilder{
+				Builder: fileAdder.CidBuilder,
+				Limit:   idHashLimit}
```

Comment:
:anger:

```go
    Limit: idHashLimit,
}
```

`go fmt` is usually pretty good about being pedantic about things like this...

Reply:
I ran go format and it was happy.  Very well I will change it.